Neos.Neos/Classes/Domain/Model/UserInterfaceMode.php

```diff
@@ -119,6 +119,24 @@ public function setFusionPath($fusionPath)
 
     /**
      * @return string
+     * @deprecated
```

Comment:
yes. I think a standard format for this annotation is

`@deprecated since 3.0`

The doc comment description my contain further hints about replacements etc.